kani-driver/src/concrete_playback.rs

```diff
@@ -383,3 +322,198 @@ mod concrete_vals_extractor {
         None
     }
 }
+
+const SPACES_4: &str = "    ";
+const SPACES_8: &str = "        ";
+
+/// Format a unit test for a number of concrete values.
+fn format_unit_test(harness_name: &str, concrete_vals: &[ConcreteVal]) -> UnitTest {
```

Comment:
Is there a reason for moving this function down here from its original position in the file? Changing its position is making it hard to see what changed in it?

Reply:
No real reason, must have been shuffled around while editing. Should I make a revision placing it back into the original place?

Reply:
Yes, I would prefer doing that to be able to easily spot what changed.